integration-tests/src/test/java/com/example/IntegrationTest.java

```diff
@@ -1,15 +1,17 @@
 package com.example;
 
-import java.util.Map;
-import java.util.Optional;
-import java.util.Set;
-import javax.inject.Provider;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.Parameterized;
 import org.junit.runners.Parameterized.Parameter;
 import org.junit.runners.Parameterized.Parameters;
 
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+
+import javax.inject.Provider;
```

Comment:
Can you revert this order change?

Reply:
Done. Can you somehow ship the code style with the project so I can just press auto-format?

Reply:
It uses https://github.com/square/java-code-styles right now (as do all my projects). I've been debating about using google-java-format but it does some nasty things to builders. I should probably relent, though.